src/models/activity.ts

```diff
@@ -125,7 +125,7 @@ export class ActivityEntity {
             let reportedAge: number | null = null
             if (typeof ageAnswer === 'string') {
               reportedAge = Number(ageAnswer)
-            } else if ('value' in ageAnswer && typeof ageAnswer.value === 'number') {
+            } else if ('value' in ageAnswer && ['number', 'string'].includes(typeof ageAnswer.value)) {
```

Comment:
Should convert ageAnswer.value to Number when it is a string

Reply:
Hey Rodrigo. I don't think I understand the reasoning for this suggestion. Why should we convert it to a number?

Reply:
Comparison bellow in line 132 will fail and reportAge variable is a Number or null

Reply:
Oh, I realise the code here is doing something slightly different than the admin (it's also missing the age range check). Thanks for pointing this out, I'll update it